### DISCUSSION TITLE
Replace Value In JSON

### DISCUSSION THREAD

**Discussion by LLCJK** [nextgenhealthcare/connect#5400] [Q&A] [answered]
I am trying to write a transformer that takes some JSON and will loop through to change the value in a specific field. My loop is below:

for(x=0; x<numEntries; x++){
	msg['entry'][x]['name'] += 'X'
}

The issue with this is that it is keeping the original value and just adding an X at the end. I need it to just say X. I am sure it is something really simple to do but I am stumped.

**Comment by pacmano1** [comment, marked as answer]
```JS
msg['entry'][x]['name'] = 'X'
```
